datalab/mlalpha/_dataset.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, files, schema=None, schema_file=None):
     elif schema_file is not None:
       with ml.util._file.open_local_or_gcs(schema_file, 'r') as f:
         self._schema = json.load(f)
+    if isinstance(files, basestring):
```

Comment:
I just made a conflicting pull request for this class.

Basically, for the structured data package, I just want the file pattern.